### PR TITLE
fix(config): change scenes sent config - fibaro walli roller

### PR DESCRIPTION
**Pull request by Micka41**

### CODE REVIEW

**Comment by Micka41** [comment]
Hello, 
What's wrong this time?

**Comment by AlCalzone** [comment]
To be fair, I have no clue yet. Something crashes during unit tests.

**Comment by AlCalzone** [comment]
Bug in Node.js 16.9:
https://github.com/nodejs/node/issues/40030

I'll have to limit testing to 16.8 for now

**Comment by Micka41** [comment]
Ok thanks.
How to restart it ?

**Comment by AlCalzone** [comment]
I'll do it once I've changed the testing.

**Comment by AlCalzone** [comment]
@Mergifyio update

File: packages/config/config/devices/0x010f/fgwreu-111.json [hunk 1]
```diff
@@ -16,132 +16,209 @@
...
-			"label": "Alarm configuration - 1st slot",
+			"label": "Alarm Configuration: 1st Slot",
 			"description": "Which alarm frames and how the device should react",
 			"valueSize": 4,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 4,
 			"defaultValue": 0
 		},
 		"31": {
```
Comment (blhoward2) [inline packages/config/config/devices/0x010f/fgwreu-111.json]: I've made some changes for you to get going, but please cleanup the remainder of the file according to our style guide. https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide

I know that these are pre-existing errors, but we appreciate your help in cleaning the file up.

File: packages/config/config/devices/0x010f/fgwreu-111.json [hunk 2]
```diff
@@ -16,132 +16,209 @@
...
 			"label": "Alarm configuration - 2nd slot",
 			"description": "which alarm frames and how the device should react",
 			"valueSize": 4,
-			"minValue": 0,
-			"maxValue": 255,
+			"minValue": 1,
+			"maxValue": 4,
```
Comment (blhoward2) [inline packages/config/config/devices/0x010f/fgwreu-111.json:129]: If there are only four options, can we define these as options so they use a drop-down? Do you know what they are?

**Review by blhoward2 — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
#3100 was just merged, so I've gone ahead and updated your file to match the new config format. As you'll see, paramInformation is now an array of objects.

I did not touch anything else, so please update the file according to the review comments

**Comment by AlCalzone** [comment]
@Micka41 are you planning to finish this?

**Pull request closed without merging**